features/step_definitions/load_map_steps.rb

```diff
@@ -0,0 +1,9 @@
+Given("I am on the home page") do
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.